docs/user_guide/datasets/acs_public_coverage.rst

```diff
@@ -0,0 +1,281 @@
+        $4209995 or more is coded as 4209995.
+
+   *  - ESR
+      - Employment status recode:
+         1. Civilian employed, at work
+         2. Civilian employed, with a job but not at work
+         3. Unemployed
+         4. Armed forces, at work
+         5. Armed forces, with a job but not at work
+         6. Not in labor force
+         
+   *  - ST
+      - State code: Please see data dictionary at `ACS PUMS documentation
+         <https://www.census.gov/programs-surveys/acs/microdata/documentation.2018.html>`_
+         for the full list of state codes.
+
+   *  - FER
+      - Gave birth to child within the past 12 months:
+         1. Yes
+         2. No
+
+   *  - RAC1P
+      - Race code
+         1. White alone
+         2. Black or African American alone
+         3. American Indian alone
+         4. Alaska Native alone
+         5. American Indian and Alaska native tribes specified; or American
+            Indian or Alaska Native, not specified and no other races
+         6. Asian alone
+         7. Native Hawaiian and Other Pacific Islander alone
+         8. Some Other Race alone
+         9. Two or More races
+
+
+The target label is given by PUBCOV, which can be used for a binary
+classification task.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 7 30
+   :stub-columns: 1
+
+   *  - Column name
+      - Description
+
+   *  - PUBCOV
+      - Public health coverage, with PUBCOV == 1 if the individual has public
+        health coverage, else 0
+
+
+.. _acs_public_coverage_discussion:
+
+Discussion
+^^^^^^^^^^
+
+As Ding et al. [1]_ notes, algorithmic (un)fairness must be evaluated in
+context of the intervention, and the effects of different interventions vary
+widely by state. Similarly, datasets should be carefully designed for the
+specific prediction task in mind.
+
+Therefore, to investigate how viable ACSPublicCoverage is as a fairness
+benchmark dataset, we must first ask what real-world tasks might require a
+prediction of whether an individual is covered by public health insurance.
+Usually, the entity that needs this information (e.g. the government checking
+for eligibilty in a social program) will simply ask the individual directly.
+Ding et al. do not provide any examples, so it is difficult to evaluate what
+"fair" even means without context. For now, the rest of this discussion focuses
+on preliminary data analysis of ACSPublicCoverage and identifies potential
+problems that users should be aware of beforehand.
+
+1. A uniform $30,000 threshold ignores differences between states.
+
+    Although the purpose of ACSPublicCoverage is to "predict whether a
+    low-income individual, not eligible for Medicare, has coverage from public
+    health insurance," this dataset should not be blindly used for all
+    prediction tasks that fall in that category. Ding et al. filters ACS PUMS
+    for all individuals whose income is below $30,000, but the 'low-income'
```

Comment:
What is meant with 'low-income' threshold here? Is this a specific threshold that is set by the government or are we referring more generally to the fact that what is "low" depends on the distribution of income, which can vary across states?

Reply:
So, the US Census Bureau does have a definition of a [poverty line](https://www.census.gov/topics/income-poverty/poverty/guidance/poverty-measures.html), but it's calculated at the household level, not individually.